### PR TITLE
Remove MarshmallowPlugin.path_helper

### PR DESCRIPTION
**Pull request by lafrech**
Addresses part of https://github.com/marshmallow-code/apispec/issues/246.

### CODE REVIEW

File: tests/test_ext_marshmallow.py [hunk 1]
```diff
@@ -232,201 +232,110 @@ def ref_path(spec):
             return '#/definitions/'
         return '#/components/schemas/'
 
+    @pytest.mark.parametrize('pet_schema', (PetSchema, PetSchema(), 'tests.schemas.PetSchema'))
```
Comment (sloria) [inline tests/test_ext_marshmallow.py:235]: 👍 Nice!

**Review by sloria — APPROVED** [review]
This looks good. It might be good to have an integration test that combines the marshmallow plugin with a framework plugin to make sure Schema names in docstrings can be parsed.

Not necessarily a blocker though. This otherwise looks good to merge. Feel free to add the integration test now or in a followup PR; I'll plan to merge this over the weekend.

**Comment by lafrech** [comment]
> It might be good to have an integration test that combines the marshmallow plugin with a framework plugin to make sure Schema names in docstrings can be parsed.

This is tested in `test_ext_order.py`. In https://github.com/marshmallow-code/apispec/issues/246 I suggest we remove this test as the order may matter, but we could keep the part that tests the right order. Basically, rename the test to "test_web_framework_and_marhsmallow" and test only [framework plugin, marshmallow] order.

**Comment by sloria** [comment]
Sounds good to me.

**Pull request merged**